scripts/build.py

```diff
@@ -1215,7 +1225,7 @@ def compile_typescript_files_continuously(project_dir):
     with open('tsc_output_log.txt', 'w') as out:
         subprocess.Popen(cmd, stdout=out)
 
-    while True:
+    while condition:
```

Comment:
Generally we try not to have variables which say "we are running this in test mode". This can cause skew between the actual code and what the test is testing.

So, it would be better for this to be `while True` and to have a test that covers it. One approach you could take is to make tsc_output_log.txt a global constant in this file. You can then swap it out during tests to be whatever you want it to be, e.g. a dummy test file which has the trigger for termination.

Reply:
Done.